starter_code/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "body-parser": "^1.18.3",
     "cookie-parser": "^1.4.3",
-    "dotenv": "^6.1.0",
+    "dotenv": "^6.2.0",
```

Comment:
no need to manually install the package as it was already installed with the irongenerator :)